### PR TITLE
Jroach/us122359

### PR DESCRIPTION
**Pull request by jeroach**
[US122359](https://rally1.rallydev.com/#/detail/userstory/456046385436?fdp=true): Unable to keyboard navigate into the file names to select them in the Evidence panel

Keyboard focus:
image

### CODE REVIEW

File: components/left-panel/consistent-evaluation-submission-item.js [hunk 1]
```diff
@@ -71,6 +71,11 @@ export class ConsistentEvaluationSubmissionItem extends RtlMixin(LocalizeConsist
 		d2l-list-item, d2l-list-item:hover {
 			--d2l-list-item-content-text-color: var(--d2l-color-ferrite);
 		}
+		d2l-list-item-content:focus {
```
Comment (jeroach) [inline components/left-panel/consistent-evaluation-submission-item.js:74]: This is the same styling used for other items with focus (drop down list, etc.)

File: components/left-panel/consistent-evaluation-submission-item.js [hunk 2]
```diff
@@ -208,6 +213,20 @@ export class ConsistentEvaluationSubmissionItem extends RtlMixin(LocalizeConsist
 		return Math.max(fileSizeBytes, 0.1).toFixed(1) + unit;
 	}
 
+	_dispatchFileSelectedKeyboardEvent(e) {
+		const fileId = e.target.getAttribute('file-id');
+
+		if (e.key === 'Enter' || e.key === ' ') {
```
Comment (jeroach) [inline components/left-panel/consistent-evaluation-submission-item.js:219]: I confirmed with Peggy that we want the navigation to trigger on either `Enter` or `Space`

**Comment by dbatiste** [comment]
I think what we really want here is the component that I have in my backlog to create - a list item whose primary action is a button, which dispatches an event.  Then there won't be any of this monkey business with manually adding tabindex or providing custom styles.  I think I've seen a couple places where list-items are being used in a button-like way, that are not keyboard accessible.

**Review by JamieTran — APPROVED** [review]
LGTM

**Review by AlexBedley — APPROVED** [review]

**Pull request merged**